wporg-gp-translation-events.php

```diff
@@ -461,7 +461,11 @@ public function generate_event_slug( array $data, array $postarr ): array {
 	 * @return void
 	 */
 	public function add_active_events_current_user(): void {
-		$user_attending_events      = get_user_meta( get_current_user_id(), Route::USER_META_KEY_ATTENDING, true ) ?: array();
+		$user_attending_events = get_user_meta( get_current_user_id(), Route::USER_META_KEY_ATTENDING, true ) ?: array();
```

Comment:
I was reviewing this problem and I saw we solved it with this approach [here](https://github.com/WordPress/wporg-gp-translation-events/blob/9e950955b446931ef738b0dfe6bf435e42b96935/includes/route.php#L130). I think it is simpler. 
```suggestion
		$user_attending_events = get_user_meta( get_current_user_id(), Route::USER_META_KEY_ATTENDING, true ) ?: array( 0 );

```

Reply:
I like the solution in this PR better because it is much more evident what is happening. I was already wondering what the `array( 0 )` is doing, it doesn't explain what's going on.

Reply:
I agree with the solution in this PR being more explicit, so I would opt for it instead or setting `array( 0 )`.

Reply:
The `arrray( 0 )` solution is returning an array with and invalid ID (0), because if you pass an empty array to the query, it returns a post :/ 
Let's merge this PR.